topics/howtosim.rst

```diff
@@ -0,0 +1,255 @@
+   gentype              : asymmetric           # DiskGenType string for velocity initialization (Jeans, Asymmetric, or Epicyclic)
+   halofile1            : XXXXX                # File with input halo model (your .model file here)
+   evolved              : false
+   hbods                : halo.bods            # Halo particle output file
+   ignore               : false                # Recompute EOF regardless of existence
+   multi                : false                # Use multimass halo
+   ndisk                : XXXXX                # Number of disk particles (e.g. 1000000)
+   ngas                 : 0                    # Number of gas particles
+   ngparam              : 3                    # Number of gas particle parameters
+   nhalo                : XXXXX                # Number of halo particles (e.g. 10000000)
+   report               : true                 # Report particle progress in EOF computation
+   runtag               : XXXXX                # Label prefix for diagnostic images (e.g. 'run1')
+   scale_height         : XXXXX                # Scale height for disk realization (e.g. 0.002)
+   scale_length         : XXXXX                # Scale length for disk realization (e.g. 0.01)
+   threads              : 1                    # Number of lightweight threads
+   zerovel              : true                 # zero center of mass and velocity
+   suffix               : ""
+
+Parameters that have values set to 'XXXXX' are those that users are encouraged to change. Example values are given in the
+comment following the parameter value. The remaining parameters are "expert parameters" and generally do not need to be changed
+unless you have a good reason to use a different value. Note that the ``exponential`` disk is
+exponential in R and :math:`sech^2` in z.
+
+To generate the initial conditions, you would run:
+
+.. code-block:: python
+
+   mpirun gendisk --config gendisk_step1.yml 
+
+in the terminal, replacing ``gendisk_step1.yml`` with the name of your YAML file that specifies the ``gendisk`` parameters. If
+you set ``ignore = false``, it may take a moment to generate the cylindrical bases cache file. The cylindrical bases will be
+saved out to whatever name is specified in ``cachefile``, and the halo basis will generally be saved out to
+``.slgrid_sph_cache``. Note that the ``diskgen`` YAML file here is distinct from the EXP YAML config file, EXP YAML config files 
+look like :ref:`this annotated example <yamlconfig>`.
+
+Step Two: generate cache files
+******************************
+If you didn't generate cache files in step one, you need to generate these files now. An example of how to do this is in the
+`pyEXP-Examples repo <https://github.com/EXP-code/pyEXP-examples/blob/main/How-To/Utilities/create%20Cylinder%20basis%20(parallel).py>`_.
+If you are including a stellar disk in your simulation, EXP requires that you provide a cylindrical cache file with a name that matches
```

Comment:
Hmmm.  I don't think that's right.  At least that's not the intent.  You do need to specify a `cachename`; that was a recent policy change.  But if the file doesn't exist, it will be rebuilt.